### ISSUE TITLE
Add Jabber/XMPP Chat Room Search Engine

### ISSUE THREAD

**Issue by GreenLunar** [searxng/searxng#772]
**Working URL to the engine**

https://search.jabber.network/

**Why do you want to add this engine?**

Libre search engine for a decentralized IM network Jabber/XMPP

**Features of this engine**

Chat Room Search Engine

**How can SearXNG fetch the information from this engine?**

Social Media

**Additional context**

Public chat room listing of the Jabber/XMPP Instant Messaging network
https://github.com/horazont/muchopper

**Comment by allendema** [comment]
More than 50% of rooms there seem to be just test rooms with under 3 users.
Just use the site itself, or search on SearXNG for the room directly.

**Comment by Bnyro** [comment]
I agree with @allendema, looking at the user counts of the largest suggested rooms it doesn't look like there are actually many users for that, and the project's GitHub repo seems abandoned too.

**Comment by GreenLunar** [comment]
> looking at the user counts of the largest suggested rooms it doesn't
> look like there are actually many users for that

Pardon myself being critic.

I suggest to do it, as long as the technology is valid.

DivestOS (a popular Android ROM), Speed Dreams (video) and Salix OS
(Linux distro) use XMPP.

You will help people to get to know XMPP or get back to it.

I encourage you to accept XMPP.

Dependency is ugly.

**Comment by sjehuda** [comment]
@allendema @Bnyro

> More than 50% of rooms there seem to be just test rooms with under 3 users.

1) This is not true.
2) Suppose it is true, the population of the "more than 50%" does not reach any of the top 100 poplar groupchats.
3) Suppose it is true, it realizes that the XMPP protocol is free and decentralized!

If we would continue to maintain such negative and disqualifying attitude, nothing will change for the better.

God forbid, if one tells to a child that he is worthless, the child and even the one who said that might believe it and the child consequently might never be as good as he could have been without dicouraging him, and I do not even takee into consideration otherwise (i.e. by encouraging him).

> I encourage you to accept XMPP.

I _urge_ you to consider and try XMPP.
